content/en/docs/setup/getting-started/index.md

```diff
@@ -94,6 +100,27 @@ Follow these steps to get started with Istio:
     ✔ Installation complete
     {{< /text >}}
 
+    {{< /tab >}}
+
+    {{< tab name="Gateway API" category-value="gateway-api" >}}
```

Comment:
That's a good point, but I don't think there's a better option.

1. I don't think we can switch the getting started page to just use the new API yet, since it's just a beta feature.
2. Just using the old API when we know that we are going to switch the default to the new API, is just getting people started with the API that we want them not to use going forward.

The approach I took is to explain that there's both APIs and let them choose, while subtly encouraging the new one by mentioning it will be the default in the future. Maybe I can improve the wording a little to make it even more clear what they're choosing?

Reply:
I would strongly prefer to not have choices on this page. This page is strictly for demos, users will have many choices ahead of them once they move beyond this page.

If @linsun is ok with this I may change my mind, as they were a major advocate of keeping this page simple.

Reply:
OK, I put back the old getting-started instructions but added a tip at the top with a pointer to the future (Gateway API) version of the doc which for now is in the additional-documents folder.